crates/composefs/src/mount.rs

```diff
@@ -78,14 +78,21 @@ pub fn erofs_mount(image: OwnedFd) -> Result<OwnedFd> {
     )?)
 }
 
-pub fn composefs_fsmount(image: OwnedFd, name: &str, basedir: impl AsFd) -> Result<OwnedFd> {
+pub fn composefs_fsmount(
+    image: OwnedFd,
+    name: &str,
+    basedir: impl AsFd,
+    enable_verity: bool,
```

Comment:
Almost feels like this should be a flags field.... in which case disable-verity would be the opt-in.  Can we make this function `pub(crate)`?  Ideally this will only ever be called via `Repository` now, which will get this logic correct...